app/src/main/java/org/pesmypetcare/mypetcare/services/googlecalendar/GoogleCalendarAdapter.java

```diff
@@ -64,6 +64,7 @@ public void deleteEvent(Pet pet, Event event) {
     @Override
     public void newSecondaryCalendar(Pet pet) {
         ExecutorService executorService = Executors.newSingleThreadExecutor();
+        System.out.println("CALENDAR TOKEN "+ pet.getOwner().getGoogleCalendarToken());
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['+' is not preceded with whitespace.](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care/pullRequest?prid=5650467)